modules/ROOT/pages/_partials/include-mule-domain-support.adoc

```diff
@@ -0,0 +1 @@
+Apps and APIs that are associated with a Mule domain appear in the Anypoint Visualizer application network, but metrics and connection information related to these apps and APIs may be inaccurate.
```

Comment:
Per the style guide, I think using **might** instead of **may** fits better here.

Reply:
```suggestion
Apps and APIs that are associated with a Mule domain appear in the Anypoint Visualizer application network, but metrics and connection information related to these apps and APIs might be inaccurate.
```